### PR TITLE
Fix direct peers

### PR DESCRIPTION
**Pull request by Menduist**
Ref https://github.com/status-im/nimbus-eth2/issues/5278

The libp2p sides fixes are here: https://github.com/status-im/nim-libp2p/pull/949

I would like to add a test to the CI to makes sure it actually works, but not sure what's the best way to do so

### CODE REVIEW

**Comment by arnetheduck** [comment]
> best way to do so

I imagine making one of the peers in the local testnet script become a direct peer and disable all of the other ways it has of getting peers is the easiest thing to do - this "should" cause the local testnet to fail if the connection is not maintained.

**Comment by Menduist** [comment]
Done, though was a bit bloody as we need to know the ENRs of two nodes before-hand.
Will merge the libp2p fix, and then this one will be ready for merge

File: docs/the_nimbus_book/src/options.md [hunk 1]
```diff
@@ -109,9 +109,10 @@ The following options are available:
      --discv5                  Enable Discovery v5 [=true].
      --dump                    Write SSZ dumps of blocks, attestations and states to data dir [=false].
      --direct-peer             The list of privileged, secure and known peers to connect and maintain the
-                               connection to, this requires a not random netkey-file. In the complete
-                               multiaddress format like: /ip4/<address>/tcp/<port>/p2p/<peerId-public-key>.
-                               Peering agreements are established out of band and must be reciprocal..
+                               connection to. This requires a not random netkey-file. In the multiaddress
+                               format like: /ip4/<address>/tcp/<port>/p2p/<peerId-public-key>, or enr format
+                               (enr:-xx). Peering agreements are established out of band and must be
+                               reciprocal..
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/options.md]:
```suggestion
                               reciprocal.
```

Comment (Menduist) [reply docs/the_nimbus_book/src/options.md]: Oh, confutils is adding a dot, funny

**Pull request merged**